tests/justice_board.test.ts

```diff
@@ -0,0 +1,80 @@
+import request from 'supertest';
+import { Express } from 'express';
+import { Collection } from 'mongodb';
+
+import { getServer } from '../index';
+import { Duty } from '../schemas/duty.zschema';
+import { Soldier } from '../schemas/soldier.zschema';
+import { createTestDutyWithId } from './data/duty.data';
+import { createTestSoldier } from './data/soldier.data';
+import { getJusticeBoard } from '../services/justice_board.service';
+import * as justiceBoardService from '../services/justice_board.service';
+import { getSoldiersCollection, getDutiesCollection } from '../db_connection';
+import { createTestSoldiersAssignedToDuties } from './data/justice_board.data';
+
+let server: Readonly<Express>;
+let soldiersCollection: Readonly<Collection<Soldier>>;
+let dutiesCollection: Readonly<Collection<Duty>>;
+
+beforeAll(async () => {
```

Comment:
I think we talked about having a describe encapsulating all tests in a file, what was your conclusion in the end?

Reply:
At the end we didn't need it to solve the problem
Should I still do this?